CMakeLists.txt

```diff
@@ -115,30 +115,11 @@ set(COGCORE_SOURCES
 
 pkg_check_modules(GIO IMPORTED_TARGET REQUIRED gio-2.0>=2.44)
 pkg_check_modules(SOUP IMPORTED_TARGET REQUIRED libsoup-2.4)
-
-# There is no need to explicitly check wpe-1.0 here because it's a
-# dependency already specified in the wpe-webkit.pc file.
-pkg_check_modules(WEB_ENGINE IMPORTED_TARGET REQUIRED wpe-webkit-1.0>=2.23.91)
-if ("${WEB_ENGINE_VERSION}" VERSION_GREATER "2.23")
-    add_definitions(-DCOG_BG_COLOR_API_SUPPORTED=1)
-else ()
-    add_definitions(-DCOG_BG_COLOR_API_SUPPORTED=0)
-endif ()
-if ("${WEB_ENGINE_VERSION}" VERSION_GREATER "2.27.3")
-    set(COG_IM_API_SUPPORTED ON)
-else ()
-    set(COG_IM_API_SUPPORTED OFF)
-endif ()
-if (COG_PLATFORM_FDO AND "${WEB_ENGINE_VERSION}" VERSION_GREATER "2.27.3")
-    set(COG_FDO_WIDGETS_SUPPORTED ON)
-else ()
-    set(COG_FDO_WIDGETS_SUPPORTED OFF)
-endif ()
+pkg_check_modules(WebKit IMPORTED_TARGET REQUIRED wpe-webkit-1.0>=2.27.3 wpe-1.0>=1.5.0)
```

Comment:
s/2.27.3/2.28.0 ?

Reply:
I guess I get the initial reasoning though, but I wouldn't really expect anyone to use pre-releases for their products.

Reply:
Development release 2.27.3 happens to be the very first one that supports all the features that this PR removes the conditional compilation guards for. I don't see a problem in leaving it like that, but also there is no downside using 2.28.0, so I'll update the PR with that 😄

Reply:
Also, I'll change it things to use `wpe-1.0>=1.6.0` and `wpebackend-fdo-1.6.0` while at it.

Reply:
Done!